src/blog/2023/11/raspberry-pi-5-flowfuse-edge-agent.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Install the FlowFuse Edge Agent on the Raspberry Pi 5"
+subtitle: "Managing your Raspberry Pi 5 with Node-RED through FlowFuse is
+easy to set up"
+description: "Managing your Raspberry Pi 5 with Node-RED through FlowFuse is
+easy to set up"
+authors: ["zeger-jan-van-de-weg"]
+date: 2023-11-07
+image: "/blog/2023/10/images/blueprints.png"
```

Comment:
Artwork?  Is this in process?

Reply:
Requested to @Yndira-FlowForge in: https://github.com/FlowFuse/customer/issues/84